XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/CameraSystem/BaseCameraDataProvider.cs

```diff
@@ -245,9 +245,9 @@ public override void Update()
             }
 
 #if !XRTK_USE_LEGACYVR
-            // We keep trying to intiailze the tracking origin,
+            // We keep trying to initialize the tracking origin,
             // until it worked, because at application launch the
-            // subystems might not be ready yet.
+            // subsytems might not be ready yet.
```

Comment:
This is still spelled incorrectly :D
Shouldn't it be subsystems?

Granted I also missed it